### DISCUSSION TITLE
Volume Increments by 1 instead of 2.

### DISCUSSION THREAD

**Discussion by TheDeeGee** [File-New-Project/EarTrumpet#1439] [Ideas]
Would be nice to control the volume in steps of 1 instead of 2 for more fine control.

**Comment by HydrantHunter** [comment]
I'm not sure if this is the same for you but, just for clarification, I'm able to adjust volume levels by increments of 1 when using the mouse to drag the sliders, but when using the mouse wheel the value changes by increments of 2.  For reference, I have my mouse wheel in Windows set to scroll only 1 line at a time.  I realize this is ultimately a request for extra work for an item of desire or convenience rather than necessity, but it would be nice if this were configurable.
